src/main/java/gov/nasa/pds/harvest/cfg/ConfigManager.java

```diff
@@ -59,8 +60,7 @@ static public List<String> exchangeLidvids (List<CollectionType> ids) {
     }
     static public String exchangeIndexForNode (String indexName) {
       if (indexNodeMap.containsKey (indexName)) return indexNodeMap.get(indexName);
-      if (0 < indexName.indexOf("-registry")) return indexName.substring(0, indexName.indexOf("-registry"));
-      return "development";
+      throw new IllegalArgumentException("Index (" + indexName + ") + not supported: either fix it in your configuration by using one of the supported or request an upgrade of harvest");
```

Comment:
Thanks @al-niessner, I would like the error message to provide the github issue link where the user can request for the upgrade needed, you can add to the message: " to support your new index by submitting a ticket on https://github.com/NASA-PDS/harvest/issues".